widgetssdk/src/main/java/com/glia/widgets/core/screensharing/data/GliaScreenSharingRepository.java

```diff
@@ -103,21 +103,21 @@ public boolean isSharingScreen() {
     }
 
     public void onDestroy() {
-        gliaCore.getCurrentEngagement().ifPresent(engagement -> {
+        gliaCore.getCurrentEngagement().ifPresent(engagement -> {//TODO move this to engagement repository
             engagement.getScreenSharing().off(
-                    ScreenSharing.Events.SCREEN_SHARING_REQUEST,
-                    screenSharingRequestConsumer
+                ScreenSharing.Events.SCREEN_SHARING_REQUEST,
+                screenSharingRequestConsumer
             );
             engagement.getScreenSharing().off(
-                    ScreenSharing.Events.VISITOR_STATE,
-                    visitorScreenSharingStateConsumer
+                ScreenSharing.Events.VISITOR_STATE,
+                visitorScreenSharingStateConsumer
             );
         });
     }
 
     private void onEngagement(Engagement engagement) {
         Logger.d(TAG, "init screen sharing");
-
+//TODO move this to engagement repository
```

Comment:
Is it a leftover or still work to be done?

Reply:
Work to be done in the next PR